packages/workbox-routing/default-export.mjs

```diff
@@ -0,0 +1,6 @@
+import Router from './lib/Router.mjs';
```

Comment:
@gauntface is this how default exports are supposed to work? I was a little unsure about how this should be set up.

Reply:
Yeah this looks like it should would.

I assume you'll want the Router publicly available too so it should be exported with `export {Router};`.

Developers could then:
```javascript
const {Router} from 'workbox-routing';
const routing from 'workbox-routing';
```

Reply:
Yes, this is what I've done for precaching and I think it works. But obviously open to alternatives as long as the end result for developers is sane.